ccd-definition/CaseEventToFields/SettleClaim-JO-nonprod.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "SETTLE_CLAIM",
+    "CaseFieldID": "partiesNames",
+    "PageID": "SettleClaim",
+    "PageLabel": "Reason for settlement",
+    "PageDisplayOrder": 1,
+    "PageFieldDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "DisplayContext": "READONLY",
+    "ShowSummaryChangeOption": "N"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "SETTLE_CLAIM",
+    "CaseFieldID": "settleReason",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "SettleClaim",
+    "PageDisplayOrder": 2,
```

Comment:
The Page Id is same, so here the PageDisplay Order should be 1 and the field, "PageFieldDisplayOrder" should be 2